docs/providers/documentation/service-now-provider.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Service Now"
+sidebarTitle: "Service Now Provider"
+description: "Service Now provider allows sending notifications and updates to Service Now."
+---
```

Comment:
Hey @rishisulakhe, thank you so much for this!
ServiceNow provider also allows one to get the topology information from ServiceNow CMDB. Can you adjust the documentation to include that as well?

Reply:
Yep, will add this